master failing on Circle?

```
    creating build/temp.linux-x86_64-2.7
    creating build/temp.linux-x86_64-2.7/c
    gcc -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -DTHREAD_STACK_SIZE=0x100000 -fPIC -DUSE__THREAD -DHAVE_SYNC_SYNCHRONIZE -I/usr/include/ffi -I/usr/include/libffi -I/usr/local/include/python2.7 -c c/_cffi_backend.c -o build/temp.linux-x86_64-2.7/c/_cffi_backend.o
    c/_cffi_backend.c:15:17: fatal error: ffi.h: No such file or directory
     #include <ffi.h>
                     ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/local/bin/python -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-6bYDIr/cffi/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-tot0Tb/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-6bYDIr/cffi/
The command '/bin/sh -c apk --no-cache update     && apk add dumb-init libstdc++ g++     && pip install --upgrade pip     && pip install --upgrade --no-cache-dir -r requirements.txt     && pip install --upgrade --no-cache-dir -r dev-requirements.txt     && apk del g++' returned a non-zero code: 1

docker build -t syncserver:build . returned exit code 1
```

https://circleci.com/gh/mozilla-services/syncserver/40 

cc @rfk?

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@ WORKDIR /app
 COPY ./requirements.txt /app/requirements.txt
 COPY ./dev-requirements.txt /app/dev-requirements.txt
 RUN apk --no-cache update \
-    && apk add dumb-init libstdc++ g++ \
+    && apk add dumb-init libstdc++ libffi-dev openssl-dev g++ \
     && pip install --upgrade pip \
     && pip install --upgrade --no-cache-dir -r requirements.txt \
     && pip install --upgrade --no-cache-dir -r dev-requirements.txt \

